project/Build.scala

```diff
@@ -32,6 +33,8 @@ object Build extends Build {
       else
         Some("releases"  at nexus + "service/local/staging/deploy/maven2")
     },
+    ScoverageSbtPlugin.ScoverageKeys.coverageMinimum := 100,
+    ScoverageSbtPlugin.ScoverageKeys.coverageFailOnMinimum := false,
```

Comment:
Master doesn't have 100% coverage right now. After fixing it, we can change this flat to `true`.